x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -1452,7 +1452,7 @@ export class Embeddable
       )
     ).forEach((dataView) => indexPatterns.push(dataView));
 
-    this.dataViews = uniqBy(indexPatterns, 'id');
+    this.internalDataViews = uniqBy(indexPatterns, 'id');
```

Comment:
what is the difference between `this.dataViews.value` and `this.internalDataViews` that we have to load here? Maybe we could just use the embeddable already existing `this.dataViews.value`?

Reply:
I can look into this in more detail, but `this.dataViews.value` is the same as calling `this.getOutput().indexPatterns`. I wasn't exactly sure why Lens had a separate `dataViews` key instead of using index patterns from the output, so I've left it to keep the behaviour the exact same.

I'm planning on cleaning this all up during the conversion of Lens to actually _use_ the new Embeddable system.